### PR TITLE
Campaign Fender Updates

### LINKED ISSUE #691: pitch page – campaign footer missing
Add "A DoSomething.org Campaign…" page region and copy.

### LINKED ISSUE #693: campaign – tighten up typography
Default typographic styles pulled from Neue do not match the header and paragraph spacing that was approved in the final mock.

### LINKED ISSUE #904: action page (all) – mobile styles
…

### LINKED ISSUE #905: pitch page (all) – mobile styles
…

### LINKED ISSUE #906: action page (do it) – layout and styles
This section is too wide. Align it to a narrower grid column and add borders to internal sections.

There is an issue with the "Do It" bottom margin collapsing, too. `overflow: auto` should fix this.

### PR DESCRIPTION
**Pull request by mmwtsn**
Two garbage plate commits to fix a handful of outstanding campaign-related fender issues.

Fixes #693
Fixes #691
Fixes #904
Fixes #905
Fixes #906

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_campaign.scss [hunk 1]
```diff
@@ -63,12 +63,17 @@
   // Wrapper for the campaign title, call to action
   // scholarship info., etc.
   div.meta {
+    width: 90%;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_campaign.scss:66]: Padding in lieu of width?

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_campaign.scss [hunk 2]
```diff
@@ -134,24 +139,47 @@
...
-    h4 {
-      font-size: $font-larger;
+    div h3,
+    div h4,
+    div p,
+    div div {
+      padding: 0 !important;
+    }
+
+    h3 {
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_campaign.scss:180]: Should we throw these into `port_to_neue.scss` so they don't get lost?

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_port-to-neue.scss [hunk 3]
```diff
@@ -1,21 +1,35 @@
 @mixin section-padding {
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_port-to-neue.scss:1]: Great use of a mixin! :+1:

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php [hunk 4]
```diff
@@ -24,4 +24,9 @@
       <?php endif; ?>
     </div>
   </header>
+
+  <footer class="boilerplate">
+    <strong>A DoSomething.org Campaign</strong>
+    <span>Join over 2.4 million young people taking action. Any Cause. Any Time. Anywhere.</span>
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php:30]: "Any Time" → "Anytime".

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php [hunk 5]
```diff
@@ -36,7 +36,9 @@
       <?php endif; ?>
 
       <?php if (isset($faq)): ?>
-      <a href="#modal-faq" class="js-modal-link">Check out our FAQs</a>
+      <ul>
+        <li><a href="#modal-faq" class="js-modal-link">Check out our FAQs</a></li>
+      </ul>
       <div id="modal-faq" class="cached-modal">
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php:42]: This guy should be in a `<script>` tag to keep it out of the DOM until necessary. Can remove the `.cached-modal` class when you do.

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_port-to-neue.scss [hunk 6]
```diff
@@ -204,3 +218,42 @@ header.header {
   display: inline-block;
   padding: 0 0.25rem;
 }
+
+// decrease base font size
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_port-to-neue.scss:222]: This file's getting hefty. Let's port promptly. :bullettrain_front:

**Comment by DFurnes** [comment]
Other than the above issues, looks smashing! :tada:

**Comment by mmwtsn** [comment]
Thanks Dave! I'm going to break those comments out into issues and work from there.

**Pull request merged**